engines/bastion_katello/test/content-hosts/content/content-host-packages-applicable.controller.test.js

```diff
@@ -51,6 +51,11 @@ describe('Controller: ContentHostPackagesApplicableController', function() {
         expect($scope.table).toBeTruthy();
     });
 
+    it("verifies controller name", function() {
+        expect($scope.controllerName).toBeTruthy();
+        expect($scope.controllerName).toBe("katello_erratum_packages");
```

Comment:
Is it the right controller?

Reply:
After our remote session, this is correct.